code/parse/sexp.cpp

```diff
@@ -3585,6 +3620,33 @@ int get_sexp()
 
 		}
 
+		// Sexp container
+		else if (*Mp == sexp_container::DELIM) {
+			Mp++;
+
+			char container_name[TOKEN_LENGTH];
+			const SCP_string container_delim(1, sexp_container::DELIM);
```

Comment:
If this is just an ad-hoc way to turn a char into a string, then why not add `DELIM_STR` as a sibling to `DELIM`?

Reply:
Good point, will fix.